ui/app/adt/app.js

```diff
@@ -53,18 +53,21 @@ angular.module('adt').config(['$stateProvider', '$httpProvider', '$urlRouterProv
             views: {
                 'header': {
                     templateUrl: 'views/headerAdt.html',
-                    controller: function ($scope) {
+                    controller: function ($scope, appService) {
                         $scope.showClinicalDashboardLink = true;
+                        $scope.enableIPDFeature = appService.getAppDescriptor().getConfigValue('enableIPDFeature');
                     }
                 },
                 'content': {
                     template: '<ui-view/>'
                 },
                 'additional-header': {
-                    templateUrl: '../common/patient/header/views/header.html'
+                    templateUrl: '../common/patient/header/views/header.html',
+                    controller: function ($scope, appService) {
+                        $scope.enableIPDFeature = appService.getAppDescriptor().getConfigValue('enableIPDFeature');
```

Comment:
Line 58 and 67 both perform the same task of initializing the scope variable. I am guessing we can do away with this, if we do a minor refactoring.